test/controllers/VerifiedEmailControllerSpec.scala

```diff
@@ -110,7 +113,92 @@ class VerifiedEmailControllerSpec extends SpecBase {
 
       when(mockEmailRepository.set(any(), any())).thenReturn(Future.successful(FailedToRetrieveEmail))
 
-      val request: FakeRequest[AnyContentAsEmpty.type] = FakeRequest(GET, getRoute)
+      val request: FakeRequest[AnyContentAsEmpty.type] = FakeRequest(GET, getVerifiedEmailRoute)
+
+      running(app) {
+        val result = route(app, request).value
+
+        status(result) mustBe INTERNAL_SERVER_ERROR
+      }
+    }
+  }
+
+  "getVerifiedEmailV2" should {
+
+    "return Not Found if no data is found in the cache and SUB09 returns no email" in new Setup {
+
```

Comment:
some of the tests in this file we go 
"abc" in new setup  { 
empty line 
test

like on this line and others we don't have an new line between. I don't mind which we do but lets be consistent if we can :)